README.md

```diff
@@ -5,6 +5,6 @@
 [![codecov](https://codecov.io/gh/etspaceman/kinesis4cats/branch/main/graph/badge.svg?token=HSJWGQ8M2W)](https://codecov.io/gh/etspaceman/kinesis4cats)
 [![Scala Steward badge](https://img.shields.io/badge/Scala_Steward-helping-blue.svg?style=flat&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAQCAMAAAARSr4IAAAAVFBMVEUAAACHjojlOy5NWlrKzcYRKjGFjIbp293YycuLa3pYY2LSqql4f3pCUFTgSjNodYRmcXUsPD/NTTbjRS+2jomhgnzNc223cGvZS0HaSD0XLjbaSjElhIr+AAAAAXRSTlMAQObYZgAAAHlJREFUCNdNyosOwyAIhWHAQS1Vt7a77/3fcxxdmv0xwmckutAR1nkm4ggbyEcg/wWmlGLDAA3oL50xi6fk5ffZ3E2E3QfZDCcCN2YtbEWZt+Drc6u6rlqv7Uk0LdKqqr5rk2UCRXOk0vmQKGfc94nOJyQjouF9H/wCc9gECEYfONoAAAAASUVORK5CYII=)](https://scala-steward.org)
 
-A comprehensive scala library for Kinesis
+A comprehensive scala library for Kinesis!!
```

Comment:
```suggestion
A comprehensive scala library for Kinesis.
```

Reply:
needs at least one exclamation point or it just comes off as low energy